editors/sc-ide/CMakeLists.txt

```diff
@@ -480,76 +482,48 @@ if(APPLE OR WIN32)
                 \"${CMAKE_INSTALL_PREFIX}/SuperCollider/SuperCollider.app\"
                 -verbose=1
                 -executable=${CONTENTS_DIR}/MacOS/sclang
                 -executable=${CONTENTS_DIR}/Resources/scsynth
-                -executable=${CONTENTS_DIR}/Resources/supernova
                 -executable=${CONTENTS_DIR}/Resources/plugins/DiskIO_UGens.scx
-                -executable=${CONTENTS_DIR}/Resources/plugins/DiskIO_UGens_supernova.scx
+            ")
+        if(SC_USE_QTWEBENGINE)
+            set(QT_WEBENGINE_APP
+            "${CONTENTS_DIR}/Frameworks/QtWebEngineCore.framework/Versions/Current/Helpers/QtWebEngineProcess.app")
+            set(QT_WEBENGINE_PROCESS "${QT_WEBENGINE_APP}/Contents/MacOS/QtWebEngineProcess")
+            string(APPEND DEPLOY_CMD "
                 -executable=${QT_WEBENGINE_PROCESS}
             ")
+        endif()
         set(VERIFY_CMD "include(BundleUtilities)")
-        if(SUPERNOVA)
-            get_target_property(LINK_FFTW libsupernova SUPERNOVA_FFTW)
-            if(LINK_FFTW)
-                message(STATUS "macdeployqt: will fix linking FFTW for supernova")
-                # For supernova, libfftw3f.3.dylib has a further dependency (libgcc_s) which is copied in,
-                # but libfftw3f is not fixed up automatically when supernova is deployed.
-                string(APPEND VERIFY_CMD "
-                message(STATUS \"Fixing up bad path in libfftw3f.3.dylib manually due to faulty logic in macdeployqt\")
-                execute_process(COMMAND install_name_tool -change /usr/local/lib/gcc/9/libgcc_s.1.dylib
-                    @loader_path/libgcc_s.1.dylib ${CONTENTS_DIR}/Frameworks/libfftw3f.3.dylib)
-                execute_process(COMMAND install_name_tool -change /usr/local/lib/gcc/10/libgcc_s.1.dylib
-                    @loader_path/libgcc_s.1.dylib ${CONTENTS_DIR}/Frameworks/libfftw3f.3.dylib)
-                execute_process(COMMAND install_name_tool -change /usr/local/lib/gcc/11/libgcc_s.1.dylib
-                    @loader_path/libgcc_s.1.dylib ${CONTENTS_DIR}/Frameworks/libfftw3f.3.dylib)
+        if("${QT_VERSION_MAJOR}" GREATER_EQUAL 6 AND SC_USE_QTWEBENGINE)
+            # Fix deployment with qt from homebrew
+            execute_process(COMMAND brew --prefix OUTPUT_VARIABLE BREW_PREFIX OUTPUT_STRIP_TRAILING_WHITESPACE)
+            get_target_property(QtCore_location Qt${QT_VERSION_MAJOR}::Core LOCATION)
+            cmake_path(IS_PREFIX BREW_PREFIX ${QtCore_location} QT_IS_FROM_HOMEBREW)
+
+            if(QT_IS_FROM_HOMEBREW)
+                message(STATUS "Detected Qt6 from homebrew: will fix deployment of QtQuickWidgets")
+                execute_process(COMMAND brew --prefix qt6 OUTPUT_VARIABLE QT_BREW_PREFIX OUTPUT_STRIP_TRAILING_WHITESPACE)
+                if(QT_BREW_PREFIX)
+                    string(APPEND VERIFY_CMD "
+                        message(STATUS \"Manually copying QtQuickWidgets due to an issue with macdeployqt\")
```

Comment:
No ticket, as this came up only in Qt6. These issues come up occasionally and it's typically finger pointing between Qt and homebrew. I found it in homebrew Qt6 build so I patched it... not sure if there's a better way to go about this.

Reply:
Maybe just leave a link to the relevant homebrew Qt6 build/patch here so in case this breaks/changes we can take a look how their code developed?

Reply:
Sorry, I meant that I found it while building SC with homebrew Qt6. There's no "patch" on homebrew/qt end, I was referring to my Cmake entry as the "patch". Macdeployqt sometimes has bugs that come out in homebrew builds, but not in the official Qt release. 
I know it would be good to report it, but that would need a clean/small reproducer, which I don't have atm.

Reply:
I think it might be desirable to try to get to the bottom of this, but I'm not sure if I'll be able to get to it for the time being. Should that block this PR or not?

Reply:
No, definitely not a blocker at all. Just if you would have some resources would have been a good place to put them there, but since there are none, there is nothing to put there.